### ISSUE TITLE
sklearn dev tests failing (deprecated argument to OneHotEncoder and QuantileTransformer)

### ISSUE THREAD

**Issue by stsievert** [dask/dask-ml#517]
`OneHotEncoder`'s `n_values` argument is deprecated pulled out in https://github.com/scikit-learn/scikit-learn/pull/13855 (and specifically [_encoders.py#L281][1]).

[1]:https://github.com/scikit-learn/scikit-learn/pull/13855/files#diff-d12408664448c94dbd880579e1b2e4d9L218

Here's some of the output from the Azure tests in #221:

tests/compose/test_column_transformer.py:38: 

``` python-traceback
>  #   def test_column_transformer():
>               (dask_ml.preprocessing.OneHotEncoder(sparse=False), ["A"]),
>         signature = {
            "n_values": n_values,
            "categorical_features": categorical_features,
            "categories": categories,
            "drop": drop,
            "sparse": sparse,
            "dtype": dtype,
            "handle_unknown": handle_unknown,
        }
>       super(OneHotEncoder, self).__init__(**signature)
E       TypeError: __init__() got an unexpected keyword argument 'n_values'
```

**Comment by stsievert** [comment]
The `QuantileTransformer` test is also failing because Scikit-Learn now uses `self._check_array(..., copy=True)` in `QuantileTransformer.fit` (see [data.py#2232]).

[1]:https://github.com/scikit-learn/scikit-learn/pull/13987/files#diff-5ebddebc20987b6125fffc893f5abc4cR2232

Here's some of the traceback:

``` python-traceback
self = <tests.preprocessing.test_data.TestQuantileTransformer object at 0x7ff5a0115860>
output_distribution = 'uniform'

    @pytest.mark.parametrize("output_distribution", ["uniform", "normal"])
    def test_basic(self, output_distribution):
        rs = da.random.RandomState(0)
        a = dpp.QuantileTransformer(output_distribution=output_distribution)
        b = spp.QuantileTransformer(output_distribution=output_distribution)
    
        X = rs.uniform(size=(1000, 3), chunks=50)
>       a.fit(X)

tests/preprocessing/test_data.py:216: 
>       X = self._check_inputs(X, copy=False)
E       TypeError: _check_inputs() got an unexpected keyword argument 'copy'
```

Dask-ML overwrites `_check_inputs`, so this version will need to have that keyword.

**Comment by stsievert** [comment]
Closed by #518
